pylint/test/extensions/test_check_raise_docs.py

```diff
@@ -351,5 +351,44 @@ def my_func(self):
         with self.assertNoMessages():
             self.checker.visit_raise(raise_node)
 
+    def test_find_missing_sphinx_raises_infer_from_instance(self):
+        raise_node = astroid.extract_node('''
+        def my_func(self):
+            """This is a docstring.
+
+            :raises NameError: Never
+            """
+            my_exception = RuntimeError('hi')
+            raise my_exception #@
+            raise NameError('hi')
+        ''')
+        node = raise_node.frame()
+        with self.assertAddsMessages(
+            Message(
+                msg_id='missing-raises-doc',
+                node=node,
+                args=('RuntimeError', ))):
+            self.checker.visit_raise(raise_node)
+
+    def test_find_missing_sphinx_raises_infer_from_function(self):
```

Comment:
OK, so in this case we would expect no error to be reported, rather than a missing-raises-doc for 'Unknown'?